style.css

```diff
@@ -0,0 +1,127 @@
+.slideShow-container, .reference, .slides-container, .btn, .Option-bar, .counter, .imgs, .imgs>img, .thumbnails-popUp, .title{
+	position: absolute;
+}
+
+.slideShow-container{
+	background-color: rgba(255,255,255,0.3);
+	height: 100%;
+	left: 0%;
+	top: 10%;
+	width: 100%;	
+}
+.slides-container{
+	height: 100%;
+	left: 0;
+	top: 0;
+	width: 100%;
+}
+.btn{
+	background-repeat: no-repeat;
+	cursor: pointer;
+	height: 100%;
+	top: 0;
+	opacity: 0.3;
+	width: 4%;
+}
+.Option-bar{
+	background-color: rgba(0,0,0,0.5);
+	height: 6%;
+	top: 94%;
+	width: 100%;
+}
+.play{
+	background-image: url(https://png.icons8.com/nolan/50/000000/circled-play.png);
+	left: 75%;
+}
+.pause{
+	background-image: url(https://png.icons8.com/nolan/50/000000/circled-pause.png);
+	left: 75%;
+}
+.prev{
+	background-image: url(https://png.icons8.com/nolan/50/000000/back.png);
+	left: 65%;
+}
+.next{
+	background-image: url(https://png.icons8.com/nolan/50/000000/forward.png);
+	left: 85%;
+}
+.thumbnails{
+	background-image: url(https://png.icons8.com/nolan/50/000000/stack-of-photos.png);
+	left: 23.5%;
+}
+.info{
+	background-image: url(https://png.icons8.com/nolan/50/000000/plus-math.png);
+	left: 10%;
+}
+.counter{
+	color: white;
```

Comment:
try to avoid using browsers colors.

why?
Because sometimes browser colors are different depending on platforms.
Let's have css variables that could be used through all the code.

Reply:
COol, Didn't know that little piece of information.